packages/markdown-docx/src/CiceroMarkToOOXMLTransformer.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Transforms the ciceromark to OOXML
+ */
+class CiceroMarkToOOXMLTransfomer {
+
+    /**
+     * Defines the different types of nodes for CiceroMark JSON
+     */
+    constructor() {
+        this.definedNodes = {
+            computedVariable: 'org.accordproject.ciceromark.ComputedVariable',
+            heading: 'org.accordproject.commonmark.Heading',
+            item: 'org.accordproject.commonmark.Item',
+            list: 'org.accordproject.commonmark.List',
+            listBlock: 'org.accordproject.ciceromark.ListBlock',
+            paragraph: 'org.accordproject.commonmark.Paragraph',
+            softbreak: 'org.accordproject.commonmark.Softbreak',
+            text: 'org.accordproject.commonmark.Text',
+            variable: 'org.accordproject.ciceromark.Variable',
+            emphasize: 'org.accordproject.commonmark.Emph',
+        };
```

Comment:
This does not need to be in a constructor. You can make this a class variable as it will be common to all objects of this class.

Reply:
@algomaster99
Should I remove the constructor completely or just declare an empty one then?

Reply:
You don't need to implement the constructor for now. Classes have a default constructor.

Reply:
@algomaster99
Done